src/nvim/window.c

```diff
@@ -749,6 +749,11 @@ win_T *win_new_float(win_T *wp, bool last, FloatConfig fconfig, Error *err)
   if (wp == NULL) {
     wp = win_alloc(last ? lastwin : lastwin_nofloating(), false);
     win_init(wp, curwin, 0);
+    // After win init new wp copied the value from curwin
+    // if new win only have one line shouldn't set wbr value
```

Comment:
Try to make comments say what something _means_, not _how_ it's done (except for cases where "how" is not obvious).

```suggestion
    // Don't inherit 'winbar' for 1-line window. #19464
```